freetextresponse/tests.py

```diff
@@ -260,6 +261,23 @@ def test_compute_score(self, credit):
             {'value': credit.value, 'max_value': Credit.full.value},
         )
 
+    @ddt.data(Credit.zero, Credit.half, Credit.full)
+    def test_compute_score_error(self, credit):
+        # pylint: disable=protected-access
+        """
+        Tests _compute_score's error handling
```

Comment:
Yes, thanks stv.  

@jlikhuva Specify exactly what's happening in the function.  We're forcing runtime.publish to throw an IntegrityError in order to test that compute_score passes it without issue.